### PR TITLE
[#164418326] resized call-to-action in the message list as 32 high leaving it at its original height in other screens

### PR DESCRIPTION
**Pull request by lmorelli986**
This PR aims to downsize call-to-action elements in the message list screen to 32px high. All other instances in other screen are left at their height.

MessageCTA in message list:
Simulator Screen Shot - iPhone 6s - 2019-03-13 at 12 57 55

==================================

MessageCTA in message details:
Simulator Screen Shot - iPhone 6s - 2019-03-13 at 12 57 58

Please note that text line height is fixed in this other PR: https://github.com/teamdigitale/io-app/pull/877

### CODE REVIEW

File: ts/components/messages/MessageCTABar.tsx [hunk 1]
```diff
@@ -133,6 +134,10 @@ class MessageCTABar extends React.PureComponent<Props, State> {
     const { message, calendarEvent, showModal, disabled } = this.props;
     const { isEventInCalendar } = this.state;
 
+    const calendarIconComponentSize = this.props.small
+      ? { width: "32", height: "32" }
```
Comment (cloudify) [inline ts/components/messages/MessageCTABar.tsx]: why we need two values if they're the same?

```suggestion
      ? 32 : 48
```

Comment (lmorelli986) [reply ts/components/messages/MessageCTABar.tsx]: changed

**Comment by cloudify** [comment]
@lmorelli986 please fix the tests

**Comment by cloudify** [comment]
@lmorelli986 I think you should really integrate prettier in your IDE, ask @fpersico 

also, pls add a screenshot to let @matteodesanti check the correctness before we merge it, thanks

**Comment by lmorelli986** [comment]
@cloudify attached screens as indicated

**Comment by cloudify** [comment]
@lmorelli986 PLEASE run the linter before pushing :)

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#164418326](https://www.pivotaltracker.com/story/show/164418326): Come CIT voglio che le call-to-action nei messaggi abbiano una altezza di 32px

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**